src/testMod/resources/data/advancedperipheralstest/computer/tests/peripheraltest.geoscanner.lua

```diff
@@ -0,0 +1,83 @@
+---
+--- Advanced Peripherals Geo Scanner tests
+--- Covers `getFuelLevel`, `getMaxFuelLevel`, `cost`,
+--- `scan`, `chunkAnalyze`, `getOperationCooldown`
+---
+
+-- TODO: replace getOperationCooldown with getScanCooldown once the function is implemented
+
+function testBlockAt(result, x, y, z, expectedName, expectedTag)
+    local blockEntry = nil
+    for _, entry in ipairs(result) do
+        if entry["x"] == x and entry["y"] == y and entry["z"] == z then
```

Comment:
I'll say that `entry.x` is better than `entry["x"]`.

Because usually, when you using `[]` operator, you are using it as a map, which means the structure of keys are unknown